Portfolio/index.html

```diff
@@ -13,18 +14,22 @@
     <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
     <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->

```

Comment:
Add a closing tag '</form>' before the line break '<br/>' for valid HTML structure.